ParabolInc/parabol
# feat(one-on-one): allow oneOnOne input in startCheckIn mutation

Fixes: #8561

**How to test:**

Enable org `oneOnOne` feature flag.

Edit this file to change variables

https://github.com/ParabolInc/parabol/blob/a3d5b02e6cea504d30136fa395b852d1d59744e0/packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx#L153-L156

1) Change variables to set unknown email, start the `one on one` meeting

```
 StartCheckInMutation(
        atmosphere,
        {gcalInput, oneOnOneTeamInput: {
            email: 'igor+unknown@parabol.co',
            orgId: 'oFJQVmaSE8'
          }},
        {history, onError, onCompleted}
      )
```

- [ ] See that a new team was created and meeting was started
- [ ] See that invite was sent

2) Accept the invite, end the meeting. Keep the same code, start a new meeting again

- [ ] See that the team was reused, no new team was created

3) Test the same cases with an email of existing user

4) See that you still can create a meeting with `teamId`
